go.sum

```diff
@@ -753,7 +753,7 @@ github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2 h1:Jamvg5psRI
 github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
 github.com/portworx/dcos-secrets v0.0.0-20180616013705-8e8ec3f66611/go.mod h1:4hklRW/4DQpLqkcXcjtNprbH2tz/sJaNtqinfPWl/LA=
 github.com/portworx/kvdb v0.0.0-20200929023115-b312c7519467/go.mod h1:Q8YyrNDvPp3DVF96BDcQuaC7fAYUCuUX+l58S7OnD2M=
-github.com/portworx/sched-ops v0.20.4-openstorage-rc3 h1:tXnHsjZT2wZ2BCXf8avDoya7zGyCgLNUC8Upt+WEQrY=
+github.com/portworx/sched-ops v0.20.4-openstorage-rc3 h1:46EZ+vYCJ3qmQolvgDCrGuPz8Tf0vIds41RuF0dqVEw=
```

Comment:
Why is this changing? I don't see anything that appears related in the code changes.

This also seems to be the reason why CI is failing. Here's what I see when I look at the CI output:

```
...
verifying github.com/portworx/sched-ops@v0.20.4-openstorage-rc3: checksum mismatch
	downloaded: h1:tXnHsjZT2wZ2BCXf8avDoya7zGyCgLNUC8Upt+WEQrY=
	go.sum:     h1:46EZ+vYCJ3qmQolvgDCrGuPz8Tf0vIds41RuF0dqVEw=

SECURITY ERROR
This download does NOT match an earlier download recorded in go.sum.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.
```

Reply:
@BlaineEXE  wrote:

> Why is this changing? I don't see anything that appears related in the code changes.

To be honest, I am not quite certain. I was puzzled myself when  I saw go.sum in the list of changed files. 🤷🏼

Here is my guess:

Earlier today, I wanted to build and unit-test locally and this was failing on go mod errors. So I tidied the dependencies and built/tested again, successfully. I assume that this has introduced the change to go.sum. 




> This also seems to be the reason why CI is failing.

Right, this seems plausible. Let me try and get rid of the go.sum change in the commit 




> Here's what I see when I look at the CI output:
> 
> ```
> ...
> verifying github.com/portworx/sched-ops@v0.20.4-openstorage-rc3: checksum mismatch
> 	downloaded: h1:tXnHsjZT2wZ2BCXf8avDoya7zGyCgLNUC8Upt+WEQrY=
> 	go.sum:     h1:46EZ+vYCJ3qmQolvgDCrGuPz8Tf0vIds41RuF0dqVEw=
> 
> SECURITY ERROR
> This download does NOT match an earlier download recorded in go.sum.
> The bits may have been replaced on the origin server, or an attacker may
> have intercepted the download attempt.
> ```

Reply:
Interestingly, when I locally revert the change to `go.sum` and run `make clean`and `go mod tidy`, I get the same error:

```console
verifying github.com/portworx/sched-ops@v0.20.4-openstorage-rc3: checksum mismatch
	downloaded: h1:46EZ+vYCJ3qmQolvgDCrGuPz8Tf0vIds41RuF0dqVEw=
	go.sum:     h1:tXnHsjZT2wZ2BCXf8avDoya7zGyCgLNUC8Upt+WEQrY=

SECURITY ERROR
This download does NOT match an earlier download recorded in go.sum.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.
```

so I am still unsure how to properly fix this.

Reply:
I have undone the change to `go.sum` and `mod-check`passes in the ci now, but locally, `make mod.check`still fails.

Reply:
@obnoxxx Can you revert this file? It seems to be causing build issues.

Reply:
done. no changes to `go.sum`in the PR anymore, even though I can't build locally without them.